### PR TITLE
feat: info page

### PR DESCRIPTION
**Pull request by Jaszkowic**
Removed HTML tags from the QA section, should be in Markdown format now. Allowed only links and paragraphs.

### CODE REVIEW

File: src/components/info/info.tsx [hunk 1]
```diff
@@ -27,12 +30,20 @@ const Info: React.FC = () => {
...
             <h1 className="text-4xl font-bold">Info</h1>
-            <ClearIcon />
+            <a
+              href="/"
+              onClick={(e) => {
+                e.preventDefault();
+                setPathname("/");
+              }}
+            >
+              <ClearIcon />
```
Comment (aeschi) [inline src/components/info/info.tsx:25]: We don't need have a close button for the info and profile page. The navigation works fully through the navbar.

File: src/components/info/qa-entry.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      <button
+        className="flex w-full flex-row justify-between text-xl"
+        onClick={() => {
+          setIsExpanded(!isExpanded);
+        }}
+      >
+        <div>{question}</div>
+        <div>
+          {isExpanded ? (
+            <ChevronDown></ChevronDown>
```
Comment (aeschi) [inline src/components/info/qa-entry.tsx:33]: these should both be in gdk-blue (color defined here https://github.com/technologiestiftung/giessdenkiez-de/pull/718)

**Review by aeschi — CHANGES_REQUESTED** [review]
already looks pretty good!

the question text style still has to be adjusted to:
    - font-semibold
    - text-2xl
 
the answer text should also be in the default color defined here #718 

Sometimes the text is centered instead of aligned to left and the seperation line is missing:
Screenshot 2024-03-07 at 15 44 46


This could be in two lines instead of three
Screenshot 2024-03-07 at 15 56 31

**Comment by Jaszkowic** [comment]
@aeschi thanks for the review, i implemented your change requests

**Comment by Jaszkowic** [comment]
still missing: logos in "Über uns" section

File: src/components/icons/chevron-right.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+const ChevronRight: React.FC = () => {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      fill="none"
+      viewBox="0 0 24 24"
+      strokeWidth={1.5}
```
Comment (aeschi) [inline src/components/icons/chevron-right.tsx:9]: small deatil but I would set the stroke-width to 2.5 for the chevron right and down

**Review by aeschi — COMMENTED** [review]
LGTM, I have two small points:

- the chevron should have stroke width 2.5
- I'm not sure if the slack button should be visible when "über das Projekt" is closed, but this would be a question for @HenrietteN

File: src/components/icons/chevron-right.tsx (hunk 3 above)
Comment (Jaszkowic) [reply src/components/icons/chevron-right.tsx]: fixed

**Comment by Jaszkowic** [comment]
@aeschi I put the Slack button in the "Über das Projekt" section, that was by mistake

**Comment by aeschi** [comment]
top! ship it 🌱

**Review by aeschi — APPROVED** [review]

**Pull request merged**